spec/integrations/allocation_spec.rb

```diff
@@ -44,11 +44,11 @@
 
```

Comment:
Git diff doesn't interpret this diff very well: I've just reduced all values by one.